config/environment.rb

```diff
@@ -1,8 +1,5 @@
 # Load the Rails application.
-require File.expand_path('../application', __FILE__)
-
-Encoding.default_external = Encoding::UTF_8
-Encoding.default_internal = Encoding::UTF_8
+require_relative 'application'
 
```

Comment:
Any idea why we don't need these?  (Or do we?)  Where did they move?

Reply:
@saracarl I don't think we need these lines, UTF-8 is the default.